sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :returns: The created or updated twin object.
+        :rtype: object
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.add(digital_twin_id, digital_twin, if_none_match="", **kwargs)
+
+    def update_digital_twin(self, digital_twin_id, twin_patch, etag=None, **kwargs):
+        # type: (str, object, str, **Any) -> object
+        """Update a digital twin using a json patch.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no digital twin with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param twin_patch: An update specification described by JSON Patch.
+            Updates to property values and $model elements may happen in the same request.
+            Operations are limited to add, replace and remove.
+        :type twin_patch: str
+        :param etag: Only perform the operation if the entity's etag matches one of
+            the etags provided or * is provided.
+        :type str
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.update(digital_twin_id, twin_patch, if_match=etag, **kwargs)
+
+    def delete_digital_twin(self, digital_twin_id, etag=None, **kwargs):
+        # type: (str, str, **Any) -> object
+        """Delete a digital twin.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no digital twin with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param etag: Only perform the operation if the entity's etag matches one of
+            the etags provided or * is provided.
+        :type str
+        :returns: None
+        :rtype: None
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twins.delete(digital_twin_id, if_match=etag, **kwargs)
+
+    def get_component(self, digital_twin_id, component_path, **kwargs):
+        # type: (str, str, **Any) -> object
+        """Get a component on a digital twin.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is either no digital twin with the provided id or the component path is
+        invalid.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param component_path: The component being retrieved.
+        :type str
+        :returns: The component object.
+        :rtype: object
```

Comment:
We should define the return type.

Reply:
It is a user defined class. We have no control on it.